dashboard/lib/constants/socket-events.js

```diff
@@ -4,4 +4,8 @@ module.exports = {
     ABORT_RUN: 'abort',
     RESUME_RUN: 'resume',
     TEST_CONN: 'test',
+
+    PAUSE_REQUEST: 'pause',
+    ABORT_REQUEST: 'abort',
+    RESUME_REQUEST: 'resume',
```

Comment:
What is the difference between `PAUSE_RUN` and `PAUSE_REQUEST`?

Reply:
Literally nothing, we are just segregating them. `PAUSE_REQUEST` signifies an event emit to notify Newman to pause the run. While `PAUSE_RUN` is the event sent by the reporter to the daemon to notify it that a run has been paused. We segregated them so if in future, we change the naming for any of the events, we won't have to change them manually across the codebase.